scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/ConstraintsTypescriptCodeGenerationService.java

```diff
@@ -0,0 +1,37 @@
+package gov.nasa.jpl.aerie.scheduler.server.services;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+import gov.nasa.jpl.aerie.constraints.TypescriptCodeGenerationService;
+import gov.nasa.jpl.aerie.merlin.protocol.types.ValueSchema;
+
+public final class ConstraintsTypescriptCodeGenerationService
```

Comment:
I think this name might be misleading because it doesn't actually generate the code.

Reply:
100% agree. It's kind of a utility class. Any preference on the name? Maybe `ConstraintsTypescriptCodeGenerationHelper`/`Util`?

Reply:
Helper sounds good

Reply:
Thanks! Done.